packages/evidence/cli.js

```diff
@@ -180,7 +183,31 @@ prog
 	.command('dev')
 	.option('--debug', 'Enables verbose console logs')
 	.describe('launch the local evidence development environment')
-	.action((args) => {
+	.action(async (args) => {
+		// Start spinner
+		const spinner = ora({
+			text: chalk.cyan('Evidence is starting\n'),
+			spinner: 'circleHalves',
+			color: 'cyan'
+		}).start();
+
+		// Set a timeout to change the spinner after 1 minute
+		const timeoutId = setTimeout(() => {
+			// Stop the current spinner
+			spinner.stop();
+
+			// Start a new spinner or modify the existing one as per your requirements
+			spinner.text = chalk.cyan(
+				'Evidence is starting - this can take up to 2 minutes the first time.'
+			);
+			spinner.start();
+		}, 30000); // 30 seconds
+
+		// Use get-port to find an available port starting from 3000
+		const port = await getPort({ port: [3000, 3001, 3002, 3003, 3004, 3005] });
```

Comment:
This feels a little fragile, probably not a huge deal (who is running 6 Vite servers on 3000)

Reply:
this falls back to a random port if none of those are available. Just ran into another scenario when playing around with this - I had 3001 occupied, but 3000 was open. So getPort picked 3000 and the proxy server picked 3001. I just fixed this up to cover that scenario